environment_py2_linux.yml

```diff
@@ -23,6 +23,7 @@ dependencies:
   - scipy>=0.16.0
   - six >=1.10.0
   - xarray>=0.10.8
+  - dask
```

Comment:
Why is dask being in the requirements (and only in linux) but not imported in the code?

Reply:
Ok, it is because of xr.mf_dataset(), right? But why only in linux?

Reply:
To be honest, I don't understand it either. I was hoping perhaps @rabernat would know more. I added it because otherwise I would get an error in xarray/backends/api.py (https://travis-ci.org/OceanParcels/parcels/jobs/441398583#L1805); but only in python3. Since Windows wasn't working anyway (#471), I didn't add it there. We could though, if we understand better

Reply:
Dask is not be *required* for xarray. It is, however, required for `open_mfdataset`.

Reply:
OK, I understand then. We use `open_mfdataset` in `example_globcurrent.py`, which we only test on linux (not on mac). We do test it on windows though, where apparently we don't need to explicitly add `dask` as a requirement?

Reply:
If it is the case, maybe dask is automatically in xarray dependencies on Windows?
But even if we don't test it on mac, we want the example to run there, so we should put dask in the mac yml files as well, no?